src/scss/tools/_transitions.scss

```diff
@@ -15,3 +15,9 @@
     opacity: 0;
   }
 }
+
+@mixin transition-ease-out(
```

Comment:
We should also update all existing occurrences, I see places in navigation-link, modal, user-info where we are transitioning color/fill, sometimes with 0.3s, sometimes with 0.5s. It would be nice to unify that.